core/src/providers/openai.rs

```diff
@@ -633,6 +628,12 @@ pub async fn streamed_completion(
     if model_id.is_some() {
         body["model"] = json!(model_id);
     }
+    if let Some(mt) = max_tokens {
+        body["max_tokens"] = mt.into();
+    }
+    if !stop.is_empty() {
```

Comment:
Aren't we changing the behavior a bit here? we should do Some(mt) ... && mt != 0 no ?

Reply:
You mean for `max_tokens`?

Reply:
Ok my bad I guess is_empty does the right thing :+1:  nvm me